Description: /usr/local/ set to /usr/
 changing file path in preferences file
Author: Thorsten Wagner (stm@users.sf.net)
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2022-05-22 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gxsm3/org.gnome.gxsm3.preferences.gschema.xml
+++ b/gxsm3/org.gnome.gxsm3.preferences.gschema.xml
@@ -1443,7 +1443,7 @@
     </key>
     <key name="gxsmpalette" type="s">
 
-      <default>'/usr/local/share/gxsm3/palette'</default>
+      <default>'/usr/share/gxsm3/palette'</default>
       <summary>searchpath to Gxsm Palette Pixmaps (1024x1 ACSII .pnm)</summary>
       <description>
         searchpath to Gxsm Palette Pixmaps (1024x1 ACSII .pnm)
@@ -1595,7 +1595,7 @@
     </key>
     <key name="palette" type="s">
 
-      <default>'/usr/local/share/gxsm3/palette/xxsmpalette.pnm'</default>
+      <default>'/usr/share/gxsm3/palette/xxsmpalette.pnm'</default>
       <summary>Path/name of your favorite palette, (1024x1 pnm RGB Bitmap (ascii))</summary>
       <description>
         Path/name of your favorite palette, (1024x1 pnm RGB Bitmap (ascii))
@@ -1611,7 +1611,7 @@
     </key>
     <key name="command1" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot1d-1.gri.sh'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot1d-1.gri.sh'</default>
       <summary>Path/name of your 1d profile plot file #1</summary>
       <description>
         Path/name of your 1d profile plot file #1
@@ -1619,7 +1619,7 @@
     </key>
     <key name="command2" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot1d-2.gri.sh'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot1d-2.gri.sh'</default>
       <summary>Path/name of your 1d profile plot file #2</summary>
       <description>
         Path/name of your 1d profile plot file #2
@@ -1627,7 +1627,7 @@
     </key>
     <key name="command3" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot1d-3.gri.sh'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot1d-3.gri.sh'</default>
       <summary>Path/name of your 1d profile plot file #3</summary>
       <description>
         Path/name of your 1d profile plot file #3
@@ -1635,7 +1635,7 @@
     </key>
     <key name="command4" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot1d-4.gri.sh'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot1d-4.gri.sh'</default>
       <summary>Path/name of your 1d profile plot file #4</summary>
       <description>
         Path/name of your 1d profile plot file #4
@@ -1643,7 +1643,7 @@
     </key>
     <key name="command5" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot1d-xmgrace.sh'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot1d-xmgrace.sh'</default>
       <summary>Path/name of your 1d profile plot file #5</summary>
       <description>
         Path/name of your 1d profile plot file #5
@@ -1651,7 +1651,7 @@
     </key>
     <key name="command6" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot1d-matplot.py'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot1d-matplot.py'</default>
       <summary>Path/name of your 1d profile plot file #6</summary>
       <description>
         Path/name of your 1d profile plot file #6
@@ -1659,7 +1659,7 @@
     </key>
     <key name="x2d-gricmdfile1" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot2d-1.gri'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot2d-1.gri'</default>
       <summary>Path/name of your 2d gri cmd file #1</summary>
       <description>
         Path/name of your 2d gri cmd file #1
@@ -1667,7 +1667,7 @@
     </key>
     <key name="x2d-gricmdfile2" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot2d-2.gri'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot2d-2.gri'</default>
       <summary>Path/name of your 2d gri cmd file #2</summary>
       <description>
         Path/name of your 2d gri cmd file #2
@@ -1675,7 +1675,7 @@
     </key>
     <key name="x2d-gricmdfile3" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot2d-3.gri'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot2d-3.gri'</default>
       <summary>Path/name of your 2d gri cmd file #3</summary>
       <description>
         Path/name of your 2d gri cmd file #3
@@ -1683,7 +1683,7 @@
     </key>
     <key name="x2d-gricmdfile4" type="s">
 
-      <default>'/usr/local/share/gxsm3/profileplot/gxsmplot2d-4.gri'</default>
+      <default>'/usr/share/gxsm3/profileplot/gxsmplot2d-4.gri'</default>
       <summary>Path/name of your 2d gri cmd file #4</summary>
       <description>
         Path/name of your 2d gri cmd file #4
